src/ansi-c/c_typecheck_expr.cpp

```diff
@@ -1943,8 +1943,46 @@ void c_typecheck_baset::typecheck_side_effect_function_call(
     if(symbol_table.symbols.find(identifier)==symbol_table.symbols.end())
     {
       // This is an undeclared function.
+
+      // Is it the polymorphic assigns clause function ?
+      if(identifier == CPROVER_PREFIX "typed_target")
```

Comment:
Why do we only check for ```typed_target``` here but not the other three build-in functions (```whole_object```, ```object_upto```, and ```object_from```)?